### PR TITLE
Adding Links on the top of readme

### PR DESCRIPTION
**Pull request by harsh9539**
@ykdojo  I added links on the top of readme #43

### CODE REVIEW

**Review by ykdojo — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# open-source
+How to get started with open source: https://www.youtube.com/watch?v=MkaIrwOlP6Y
```
[the remark this answers is not in the document]
Comment (harsh9539) [reply README.md:2]: Ok sure

**Comment by harsh9539** [comment]
Added under the main Header. #44

**Comment by harsh9539** [comment]
Hii @ykdojo Please check this is it correct now. #44

File: README.md [hunk 2]
```diff
@@ -7,6 +7,11 @@ A list of friendly active repos for contributing!
 ---
 
 # List
+# open-source
```
Comment (ykdojo) [inline README.md:10]: This should not be part of the list.

Can you try to be more careful with checking everything before you send updates / PRs?

**Review by ykdojo — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -6,8 +6,18 @@ A list of friendly active repos for contributing!
...
+How to get started with open source: https://www.youtube.com/watch?v=MkaIrwOlP6Y
+
+
+# defaang
+defaang / dojo clan Discord (in case people want to ask questions or chat about open source): https://discord.com/invite/nNtVfKddDD
+---
+
 # List
 
+
```
Comment (ykdojo) [inline README.md:19]: Not sure why these blank lines have been added

File: README.md [hunk 4]
```diff
@@ -6,8 +6,18 @@ A list of friendly active repos for contributing!
 
 ---
 
+# open-source
+How to get started with open source: https://www.youtube.com/watch?v=MkaIrwOlP6Y
+
+
+# defaang
+defaang / dojo clan Discord (in case people want to ask questions or chat about open source): https://discord.com/invite/nNtVfKddDD
+---
```
Comment (ykdojo) [inline README.md:15]: You might need to add a blank line before this line for the horizontal line to show up correctly

File: README.md [hunk 5]
```diff
@@ -6,8 +6,18 @@ A list of friendly active repos for contributing!
```
Comment (ykdojo) [inline README.md:6]: I think the cleanest way to add these would be as part of this list. Can you make that change?

**Review by ykdojo — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (harsh9539) [reply README.md:6]: Done sir

**Comment by harsh9539** [comment]
Done sir

**Pull request closed without merging**